js/background/inject/inject.js

```diff
@@ -103,8 +103,7 @@ $j(document).ready(function () {
                 showPasswordPicker(form);
             }
         }
-
-        // $j(el).bind('click', onClick);
+        $j(el).unbind('click', onClick);
```

Comment:
This does not work. onClick is not actually unbound because the function is recreated each time and creates a different reference. Need to pull onClick out of createdFormIcon